ci-operator/step-registry/ipi/install/install/gcp/on-bastionhost/ipi-install-install-gcp-on-bastionhost-ref.yaml

```diff
@@ -0,0 +1,46 @@
+ref:
+  as: ipi-install-install-gcp-on-bastionhost
```

Comment:
I think this step is similar to https://github.com/openshift/release/pull/48193#issuecomment-1922782992, suggest to reuse that step to run gcp install in bastion.